test/mnist_gpu.jl

```diff
@@ -57,7 +57,7 @@ m = Flux.Chain(down,
     DiffEqArray_to_Array,
     fc) |> gpu

```

Comment:
```suggestion
    fc) |> Flux.gpu

```

Reply:
Also here

https://github.com/SciML/DiffEqFlux.jl/blob/e019eb799e80cf0ad9e4d55766013c4d757c1e63/test/mnist_gpu.jl#L50

Reply:
I think its happening because we have `using Lux` here instead of the `import Lux` in master . But we can resolve these namespace issues by just depending on `LuxCore` since the layer types are all we need from Lux

Reply:
Yeah in DiffEqFlux we should probably only use LuxCore.